### PR TITLE
Changes to Analize Sentiment model + renamings

### LINKED ISSUE #9688: Change `SentimentClass` to something less confusing
In the UX study, `SentimentClass` was confusing and perceived as a Class, not classification.

### PR DESCRIPTION
**Pull request by maririos**
Fixes: #9688

### CODE REVIEW

File: sdk/core/Azure.Core/tests/TestFramework/LiveOnlyAttribute.cs [hunk 1]
```diff
@@ -25,6 +25,7 @@ public void ApplyToTest(Test test)
             if (test.RunState != RunState.NotRunnable)
             {
                 RecordedTestMode mode = RecordedTestUtilities.GetModeFromEnvironment();
+                mode = RecordedTestMode.Live;
```
Comment (pakrym) [inline sdk/core/Azure.Core/tests/TestFramework/LiveOnlyAttribute.cs]: ??

**Review by pakrym — APPROVED** [review]

File: sdk/core/Azure.Core/tests/TestFramework/LiveOnlyAttribute.cs (hunk 1 above)
Comment (maririos) [reply sdk/core/Azure.Core/tests/TestFramework/LiveOnlyAttribute.cs]: Thank you!!! left after some testing. Reverting ....

File: sdk/textanalytics/Azure.AI.TextAnalytics/src/TextSentimentScores.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// The sentiment confidence scores, by sentiment class label.
+    /// </summary>
+    public class TextSentimentScores
```
Comment (kristapratico) [inline sdk/textanalytics/Azure.AI.TextAnalytics/src/TextSentimentScores.cs]: Is `SentimentConfidenceScorePerLabel` not a good name here? Forgive me if I'm confusing the types again.

**Review by kristapratico — APPROVED** [review]

File: sdk/textanalytics/Azure.AI.TextAnalytics/src/TextSentimentScores.cs (hunk 2 above)
Comment (maririos) [reply sdk/textanalytics/Azure.AI.TextAnalytics/src/TextSentimentScores.cs]: the property the user calls is `SentimentScores`, so for me, `TextSentimentScores` sounds good.
`SentimentConfidenceScorePerLabel ` is super explicit on what it is, which I like, but seems too long.
Will bring up for discussion with other languages

Comment (kristapratico) [reply sdk/textanalytics/Azure.AI.TextAnalytics/src/TextSentimentScores.cs]: Could shorten it to `SentimentScorePerLabel`.

**Pull request merged**